cmd/notation/registry.go

```diff
@@ -14,29 +16,29 @@ import (
 	"oras.land/oras-go/v2/registry/remote/auth"
 )
 
-func getSignatureRepository(opts *SecureFlagOpts, reference string) (notationregistry.Repository, error) {
+func getSignatureRepository(opts *SecureFlagOpts, reference string, debug bool) (notationregistry.Repository, error) {
```

Comment:
having parameter explicitly for debug doesn't looks right. Can we expose a new method like setDebug and use that to set the required flags?

Reply:
I agree with @priteshbandi , that's mostly the standard way to do it.

Reply:
Update. Did a litter refactoring to expose remote.Reposity for updating the debug setting.